wallet/local/wallet.go

```diff
@@ -27,21 +27,37 @@ import (
 
 // LocalWallet represents a local wallet implementation
 type LocalWallet struct {
-	*primary.Wallet
-	accounts []account.Account
+	wallet            *primary.Wallet            // Primary wallet for P/X/C operations
+	accounts          map[string]account.Account // Named accounts map
+	activeAccountName string                     // Currently active account name
+	defaultNetwork    network.Network            // Default network for operations
```

Comment:
do we need defaultnetwork since we'll only have one network at a time? maybe just netowrk?

Reply:
Good point - renamed to `activeNetwork` for consistency with `activeAccountName`. Both are settable via `SetActiveAccount()` and `SetNetwork()` respectively.